.mage/go.go

```diff
@@ -96,7 +96,7 @@ func (Go) packageDirs() (packageDirs []string, err error) {
 		goPackageDirs = packageDirs
 	}()
 
-	dirs, err := outputGo("list", "-f", "{{.Dir}}", "./...")
+	dirs, err := outputGo("list", "-f", "{{.Dir}}", "./pkg/...", "./cmd/...")
```

Comment:
But `.` and `.mage` (and later `sdk`) also contain Go packages

Reply:
Left this as it was and did https://github.com/TheThingsNetwork/lorawan-stack/pull/1312/commits/fc92a079a7c1885408a40ff394c7f95115ac152c instead to fix the `misspell` target